### ISSUE TITLE
Duplicate calendar entries rather than two calendars?

### ISSUE THREAD

**Issue by snigehere** [jampez77/DVLA-Vehicle-Enquiry-Service#18]
When I configured the integration I thought I set two calendars in the config to add the reminder.  I clicked configure against the entry  to verify this but could only change the scan time in that dialogue, no option to change the calendars. Can you tell me where this detail is stored so I can verify? 

I ask because what I have is two entries for each car and for each event (tax and mot are different dates) in one calendar and none in the other.  It maybe that these are left over from my re-install a couple of times to change the calendar configuration as I didn't clean out the calendar ahead of re-install.

Thanks

**Comment by jampez77** [comment]
I have added the listed calendars in the latest release [2024.9.2](https://github.com/jampez77/DVLA-Vehicle-Enquiry-Service/releases/tag/2024.9.2). It's just an experiment for this issue, by which i mean that I haven't done anything beyond just that.

Selecting a new calendar `should` add events to it but at this stage, unselecting a calendar probably won't do anything.

Let me know how you get one.

**Comment by snigehere** [comment]
Many thanks
I now see the calendars I selected in the config which I think is a great addition.  I am OK with nothing happening if I turn off a selected calendar .. I would expect to have to delete any entries manually anyway

Using this info I have been able to clean up some old entries in other calendars which I guess got created when I was messing about testing so I now have a clean setup.  The two missing entries got created after the restart following the version upgrade so that solved that as well.

Thanks for your help
